packages/vite/CHANGELOG.md

```diff
@@ -1,9 +1,26 @@
 ## 4.1.0 (2023-02-02)
 
+Vite 4.1 updates to the latests version of Rollup and esbuild. Check out the new [Rollup docs](https://rollupjs.org/), that are now powered by VitePress making the navigation between Vite and Rollup docs easier for users.
 
+[Vite docs](https://vitejs.dev) got a theme update migrating to the latest version of VitePress.
 
+As part of [Vite 4](https://vitejs.dev/blog/announcing-vite4.html), the Vue and React plugins have been extracted out of the monorepo. Although their release cycle will no longer follow Vite releases moving forward, Vite 4.1 is released in parallel with new versions of [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/releases/tag/plugin-react%403.1.0) and [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react-swc/releases/tag/v3.1.0). @vitejs/plugin-react 3.1.0 reworks the way HMR is handled fixing many edge cases.
```

Comment:
Should we add a quick one line mention for the support of SWC plugins?

Reply:
Totally. Would you add a suggestion?